modules/nf-core/minimac4/impute/main.nf

```diff
@@ -0,0 +1,67 @@
+process MINIMAC4_IMPUTE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/minimac4:4.1.6--hcb620b3_1':
+        'biocontainers/minimac4:4.1.6--hcb620b3_1' }"
+
+    input:
+    tuple val(meta), path(target_vcf), path(target_index), path(ref_msav), path(sites_vcf), path(sites_index), path(map)
+
+    output:
+    tuple val(meta), path("*.{bcf,sav,vcf.gz,vcf,ubcf,usav}"), emit: vcf
```

Comment:
Wouldn't one optional channel per format be more robust?

Reply:
This is one of the current issue with many modules in nf-core.
After checking the guidelines, it seems that the aggregated should be preferred:
https://nf-co.re/docs/guidelines/components/modules#one-output-channel-per-output-file-type